spec/java/util/HashMap.lsl

```diff
@@ -1,45 +1,41 @@
-libsl '1.1.0';
+libsl "1.1.0";
 
-library 'std:collections' language 'Java' version '11' url '-';
+library "std:collections" language "Java" version "11" url "-";
 
-import 'java-common.lsl';
-import 'list-actions.lsl';
+// imports
 
+import "list-actions.lsl";
 
+import "java-common.lsl";
+import "java/util/interfaces.lsl"
+import "java/util/function/interfaces.lsl"
 
-@Alias('java.util.Set')
-typealias Set = Object;    // #problem
 
-@Alias('java.util.Map')
-typealias Map = Object;    // #problem
+@TypeMapping("java.util.ObjectInputStream")
+typealias ObjectInputStream = Object;  // #problem
 
-@Alias('java.util.function.Function')
-typealias Function = Object;    // #problem
-
-@Alias('java.util.function.BiFunction')
-typealias BiFunction = Object;    // #problem
-
-@Alias('java.util.function.BiConsumer')
-typealias BiConsumer = Object;    // #problem
 
+// automata
 
 @Public
-@Extends('java.util.AbstractMap')
+@Extends("java.util.AbstractMap")
 @WrapperMeta(
-    src='java.util.HashMap',
-    dst='org.utbot.engine.overrides.collections.UtHashMap',
-    matchInterfaces=true,
+    src="java.util.HashMap",
+    dst="ru.spbpu.libsl.overrides.collections.HashMap",
+    forceMatchInterfaces=true,
 )
 automaton HashMap: int
 (
-    var keys: list<Object>;
-    var values: list<Object>;
-    var length: int;
-    @Transient var modCounter: int;
+    var keys: list<Object> = null;
+    var values: list<Object> = null;
+    var length: int = 0;
```

Comment:
If the serialization procedure will be described in specification, then, I suspect, we would have difficulties re-creating state: the lengths of lists would not be accessible. So it would be useful to leave it searializable, I guess.